### ISSUE TITLE
FormsModule Not working

### ISSUE THREAD

**Issue by Abdul-majid-ashrafi** [maciejtreder/angular-universal-pwa#189]
## I'm submitting some data but ngValue,ngModel not working here is my code

code home-component.ts
<pre><code>
  import { NgForm } from '@angular/forms';
</code></pre>

code home-component.html
<pre><code>
    < div class="col-md-2">
        < select *ngIf="segment.breathCycle.length > 1" [(ngModel)]="segment.selectedDuration">
         < option *ngFor="let d of segment.breathCycle;" [ngValue]="d">{{d.breathCycleDuration}}<option>
              < /select>
            < /div>
</code></pre>

code app.modulet.ts
<pre><code>
import { FormsModule, ReactiveFormsModule } from '@angular/forms';

@NgModule({
    declarations: [
        AppComponent,
        NavComponent
    ],
    imports: [
        MatButtonModule,
        MatSnackBarModule,
        MatMenuModule,
        TranslateModule.forChild(),
        CommonModule,
        HttpModule,
        FormsModule,
        RouterModule.forRoot([
            // { path: '/', component: DashboardComponent },
            { path: 'home', component: DashboardComponent }
        ]),
        HttpClientModule,
        PrebootModule.withConfig({ appRoot: 'app-root' })
    ],
    providers: [SnackBar, WindowRef,HttpService, Notifications, HitWithTransferStateResolver, HitWithoutTransferStateResolver, ExampleApi, Title, Meta],

    bootstrap: [AppComponent]
})
</code></pre>

I am following this 
<pre><code>
npm install -g angular-universal-pwa
ng new --collection angular-universal-pwa myApp
cd myApp
npm start
</code></pre>

### I was start today. When I run npm start It's working good, but when I run npm run build:prod for server I am getting error Please Solve this
Thank you
## Error
![capture](https://user-images.githubusercontent.com/10865465/38376005-0473ccc8-3911-11e8-9cf4-22f486cb0de1.PNG)

**Comment by maciejtreder** [comment]
@Abdul-majid-ashrafi 

Thank you for this report.

Unfortunately, I am unable to reproduce this issue. The only thing which I found is a problem with `action-xhr` which comes from AMP. But! The form is working on my end (vide attached commit: https://github.com/maciejtreder/angular-universal-pwa/commit/dfeddfad7b67ce8e9d13917a0ea525abb0e9f82e).

The only thing which I did is remove AMP stuff.

Could try to reproduce this issue on the latest build (1.1.2)? I used this one to reproduce your issue.

To update just type:

```
npm update -g angular-universal-pwa
ng new --collection angular-universal-pwa myApp
```
